src/component/project/Project/DeleteProject/DeleteProjectDialogue.tsx

```diff
@@ -0,0 +1,53 @@
+import { Dialogue } from '../../../common/Dialogue/Dialogue';
+import React from 'react';
+import { formatUnknownError } from '../../../../utils/formatUnknownError';
+import useProjectApi from '../../../../hooks/api/actions/useProjectApi/useProjectApi';
+import useProjects from '../../../../hooks/api/getters/useProjects/useProjects';
+import useToast from '../../../../hooks/useToast';
+
+interface IDeleteProjectDialogueProps {
+    project: string;
+    open: boolean;
+    onClose?: (e: React.SyntheticEvent, reason?: string) => void;
+    onFinish?: () => void;
+    onSuccess?: () => void;
+}
+
+export const DeleteProjectDialogue = ({
+    open,
+    onClose,
+    project,
+    onFinish,
+    onSuccess,
```

Comment:
Having:
 - `onClose`
 - `onFinish`
 - `onSuccess`
 
Seems a bit confusing at first sight. Might be worth having a second look to check whether we can simplify this.

Reply:
Removed onFinish